solidity/test/liquidation-tests/redemptionProofTimeoutTest.js

```diff
@@ -35,7 +35,7 @@ describe("Integration -- Redemption-proof timeout", async function () {
     it("liquidates correctly", async () => {
      await toAwaitingWithdrawalProof(testDeposit)
       //  AWAITING_WITHDRAWAL_SIGNATURE -> AWAITING_WITHDRAWAL_PROOF
-      const requirement = await testDeposit.getOwnerRedemptionTbtcRequirement.call(depositInitiator)
+      const requirement = await testDeposit.getRedemptionTbtcRequirement.call(depositInitiator)
```

Comment:
Why are we expecting on this value below at all? It seems irrelevant to the test, no? Ditto for the two below.